src/quicr_client.cpp

```diff
@@ -124,6 +145,7 @@ class QuicRTransportDelegate : public ITransport::TransportDelegate
 
 void
 QuicRClient::make_transport(RelayInfo& relay_info,
+                            qtransport::TransportConfig tconfig,
```

Comment:
Should TransportConfig be marked as &&?

Reply:
In terms of performance, this is not an issue. A copy should be fine. We cannot use ```std::move()``` considering the caller can/will reuse the config for multiple instances of the client/server. Laps is an example of this where it uses the same config and passes it with initial settings to the first server instance and then minor changes to the second (e.g., port and TLS cert changes).